handlers.go

```diff
@@ -4203,6 +4204,11 @@ func (s *Server) handleColfsListDir(c echo.Context, u *User) error {
 			return err
 		}
 
+		// trying to list a CID dir, not allowed
+		if strings.HasPrefix(relp, ".") && r.Dir == true {
+			return fmt.Errorf("listing CID directories is not allowed")
```

Comment:
just to verify .. this will fail the entire listing if a CID directory is a child of an existing collection? is there a middle ground between the CID dir being the same as `relp` and being a child of `relp` ?

Reply:
The relative path `relp` is checked against `dir` from the request. `relp` will only be `.` if `dir` and `r.Path` are exactly the same (aka the request is `collections/fs/list?col=uuid&dir=/path/to/cid/dir`)